index.html

```diff
@@ -41,6 +40,151 @@ <h1>I'm Tash <br> Glad to see you!</h1>
+			</div>
+			<p class="day">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs">
+				<button class="btn1" type="submit">html</button>
+				<button class="btn2" type="submit">css</button>
+				<button class="btn3" type="submit">javaScript</button>
+			</div>
+			<button class="btn4" type="submit">See Project</button>
+			</div>
+
+			<div class="works2">
+			<img class="snap2" src="images/Snap4.png" alt="">
+			<p class="tonic2">Multi-Post Stories</p>
+
+			<div class="grid-container2">
+				  <div class="grid-item12"><p>CANOPY</p></div>
+				  <div class="grid-item22"><p>.</p></div>
+				  <div class="grid-item32"><p>Back End Dev</p></div>
+				  <div class="grid-item42"><p>.</p></div>
+				  <div class="grid-item52"><p>2015</p></div>
+			</div>
+			<p class="day2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs2">
+				<button class="btn12" type="submit">html</button>
+				<button class="btn22" type="submit">css</button>
+				<button class="btn32" type="submit">javaScript</button>
+			</div>
+			<button class="btn42" type="submit">See Project</button>
+			</div>
+
+			<div class="works3">
+			<img class="snap3" src="images/Snapshoot3.png" alt="">
+			<p class="tonic3">Tonic</p>
+
+			<div class="grid-container3">
+				  <div class="grid-item13"><p>CANOPY</p></div>
+				  <div class="grid-item23"><p>.</p></div>
+				  <div class="grid-item33"><p>Back End Dev</p></div>
+				  <div class="grid-item43"><p>.</p></div>
+				  <div class="grid-item53"><p>2015</p></div>
+			</div>
+			<p class="day3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs3">
+				<button class="btn13" type="submit">html</button>
+				<button class="btn23" type="submit">css</button>
+				<button class="btn33" type="submit">javaScript</button>
+			</div>
+			<button class="btn43" type="submit">See Project</button>
+			</div>
+
+			<div class="works4">
+			<img class="snap4" src="images/Snapshoot4.png" alt="">
+			<p class="tonic4">Multi-Post Stories</p>
+
+			<div class="grid-container4">
+				  <div class="grid-item14"><p>CANOPY</p></div>
+				  <div class="grid-item24"><p>.</p></div>
+				  <div class="grid-item34"><p>Back End Dev</p></div>
+				  <div class="grid-item44"><p>.</p></div>
+				  <div class="grid-item54"><p>2015</p></div>
+			</div>
+			<p class="day4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs3">
+				<button class="btn14" type="submit">html</button>
+				<button class="btn24" type="submit">css</button>
+				<button class="btn34" type="submit">javaScript</button>
+			</div>
+			<button class="btn44" type="submit">See Project</button>
+			</div>
+
+		</section>
+
+		<div class="div-class">
+
+		<section class="about-myself">
+
+      <div class="desk-about">
+        <h1 class="aboutmy">About <br> Myself</h1>
```

Comment:
- [ ] Kindly ensure that your content is correctly indented, the rest of your code looks really clean but the indentation in the about section could definitely be improved. Using a code formatting extension like [prettier](https://prettier.io/) can help you out.